### DISCUSSION TITLE
Welcome to Open Source Yoot Tower Discussions!

### DISCUSSION THREAD

**Discussion by SimHacker** [SimHacker/YootTower#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SimHacker** [comment]
Hi!!!

**Reply by Danny500** [reply]
HI 

*Waves in Forrest Gump!

**Comment by DDaveson** [comment]
So I've been keeping my eye on this project for a while after my initial excitement in finding it. I'm not seeing much going on, is it dead?
